src/cswinrt/main.cpp

```diff
@@ -212,6 +217,20 @@ Where <spec> is one or more of:
                                         write_factory_class(w, type);
                                     }
                                 }
+                                // Event source code generation:
```

Comment:
consider moving this into a write_class_event_source_subclass near related code in code_writers.h, and implement it in terms of write_interface_event_source_subclass below